RULES.md

```diff
@@ -0,0 +1,3 @@
+# Rules for translators
+
+See the following wiki article about translation rules: [wiki.integreat-app.de/translations-internal](https://wiki.integreat-app.de/translations-internal)
```

Comment:
Why not add this to the readme?

Reply:
To have it separate to be able to provide it to external translators.